### PR TITLE
Remove unnecessary test metaclass

### PR DESCRIPTION
**Pull request by bmw**
Fixes mypy's complaints.

### CODE REVIEW

File: certbot-apache/certbot_apache/tests/http_01_test.py [hunk 1]
```diff
@@ -137,6 +116,31 @@ def test_no_vhost(self):
         self.config.config.http01_port = 12345
         self.assertRaises(errors.PluginError, self.http.perform)
 
+    def test_perform_1_achall_22(self):
```
Comment (ohemorange) [inline certbot-apache/certbot_apache/tests/http_01_test.py]: you've got some strange extra 2s hanging around in here

Comment (ohemorange) [reply certbot-apache/certbot_apache/tests/http_01_test.py]: oh maybe you want the whole version in there

Comment (bmw) [reply certbot-apache/certbot_apache/tests/http_01_test.py]: > oh maybe you want the whole version in there

That was the intention, but this suggests I can make it more clear. Merging this PR now to try and make sure you're not blocked on it, but I'll write a follow up PR.

Comment (bmw) [reply certbot-apache/certbot_apache/tests/http_01_test.py]: Done in #5865.

**Pull request merged**